### ISSUE TITLE
Adjust degree IV in C major just edo 53 scale in third

### ISSUE THREAD

**Issue by mbarbin** [mbarbin/fingerboard#1]
As seen in [systems/cello/e53/thirds.ml](https://github.com/mbarbin/fingerboard/tree/main/systems/cello/e53/thirds.ml), the interval of the minor third shown below is 13-53edo. I intend for it to be adjusted to 14-53edo like the rest of the minor thirds in this table. This would be by making use of one of the [Zarlinean combination](https://github.com/mbarbin/fingerboard/tree/main/src/zarlinean_combination.ml) with the [changing degree on the F](https://github.com/mbarbin/fingerboard/tree/main/test/test__zarlinean_combination.ml) (degree IV) : specifically by adjusting the F3 to m3z-e53. 
```
┌─────┬────────┬───────────┬───────┬──────┬────────┬───────────┬───────┬───────────────┬───────┐
│ Low │ String │ Pos       │ Cents │ High │ String │ Pos       │ Cents │ Interval      │ Cents │    
├─────┼────────┼───────────┼───────┼──────┼────────┼───────────┼───────┼───────────────┼───────┤
│ D3  │ III    │ 5p-e53    │ 702   │ F3   │ II     │ m3p-e53   │ 294   │ m3 - 13-53edo │ 294   │
└─────┴────────┴───────────┴───────┴──────┴────────┴───────────┴───────┴───────────────┴───────┘
```
I haven't determined yet whether to do this systematically as part of the scale constructor, or whether this should be overridden manually. I need to spend some time thinking about whether there exists a principal criteria that would lead to that particular choice for the changing degree. I intuitively suspect that such criteria exists.

**Comment by mbarbin** [comment]
Note that the same is happening in sixths, which are built with the same code path. I suspect that the same fix will solve the issues in both thirds and sixths.

**Comment by mbarbin** [comment]
Fixed by 039c4305fa4b02a5c6a87a1f5be3e90ffd53d4f9.
